.travis.yml

```diff
@@ -7,9 +7,9 @@ python:
 # command to install dependencies
 install:
   - pip install -r requirements.txt
-  
+
 # command to run tests
 script: pytest
```

Comment:
if there are no tests then just remove these for now (inc. coveralls statement)
Travis will then pass if (minimally) the project installs